.github/issue_template.md

```diff
@@ -0,0 +1,27 @@
+### If it's a discussion, please start one [here](https://github.com/cultureamp/kaizen-design-system/discussions).
+
+- [ ] Before opening this issue, I've searched open and closed issues for a fix.
+
+## Expected Behavior
+
+
+## Actual Behavior (please provide screenshots)
```

Comment:
Some of our issues are code only, should we clarify that? 

```suggestion
## Actual Behavior (please provide screenshots if it's a visual issue)
```

Reply:
Yep I like that.